packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -64,7 +64,8 @@ export const CaptionsBanner = (props: { isMobile: boolean }): JSX.Element => {
       }
       {
         /* @conditional-compile-remove(close-captions) */ captionsBannerProps.captions.length > 0 &&
-          captionsBannerProps.isCaptionsOn && (
+          captionsBannerProps.isCaptionsOn &&
+          !props.disableBannerForHoldScreen && (
```

Comment:
nit.  Double negatives can trip anyone up.

Can we change the prop to be showBanner and then we can pass false into it if localHold is true
then we can more easily say  isCaptionsOn && showBanner && (....

Reply:
Agree, we can have a property named "isOnHold" to disable the captions banner, or more straightforward, showBanner

Reply:
disableBannerForHoldScreen is consistent with the name disableButtonsForHoldScreen which we pass into the control bar for a similar purpose. 
Should we keep this naming pattern or just pass in something like showBanner/ isOnHold?

Reply:
Changed name to hideBanner, makes more sense than disablebanner since we are hiding the entire UI.
It's still a double negative but easier to understand compared to disableBanner, also in this case when we conditionally compiled this line out for PSTN, the prop will be of value 'defined' which equals to false. (We don't need to have extra logic to set default value)